codeflash/cli_cmds/cmd_init.py

```diff
@@ -802,6 +805,37 @@ def install_github_actions(override_formatter_check: bool = False) -> None:  # n
         apologize_and_exit()
 
 
+def install_vscode_extension() -> None:
+    vscode_path = shutil.which("code")
+    if not vscode_path:
+        return
+
+    error = ""
+    with progress_bar("Installing Codeflash for VSCode…"):
+        try:
+            result = subprocess.run(
+                [vscode_path, "--install-extension", "codeflash.codeflash", "--force"],
+                check=True,
+                text=True,
+                timeout=60,
+                capture_output=True,
+            )
+        except subprocess.TimeoutExpired:
+            error = "Installation timed out."
+        except subprocess.CalledProcessError as e:
+            error = e.stderr or "Unknown error."
+
+    if error:
+        click.echo("❌ Failed to install Codeflash for VSCode.")
```

Comment:
Rather than failed msg, we can suggest user and error msg can go to our telemetry for our own logs.